test/web-platform-tests/to-upstream/html/webappapis/animation-frames/no-interference-timers.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>AnimationTiming Test: does not interfere with timers</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-window-requestanimationframe">
```

Comment:
no idea what metadata should be here. I just copied a test from the existent animation-frame tests